compiler/plugins/target/ROCM/builtins/ukernel/compile_flags.txt

```diff
@@ -0,0 +1,15 @@
+# Local clangd configuration file for this directory.
+# The flags here mirror CMakeLists.txt. Flags that should not matter to clangd
+# have been omitted.
+
+-x c
+-std=c23
+-I
+../../../../../..
```

Comment:
What is this relative to?

Reply:
To IREE_SOURCE_DIR. So that `#include "compiler/plugins/target/ROCM/builtins/ukernel/common.h" works.